server/src/main/java/online/dipa/hub/TimelineState.java

```diff
@@ -0,0 +1,81 @@
+package online.dipa.hub;
+
+import online.dipa.hub.api.model.Milestone;
+import online.dipa.hub.api.model.Task;
+
+import java.time.LocalDate;
+import java.util.List;
+
+public class TimelineState {
```

Comment:
Ich hätte die Attribute der Timeline gar nicht direkt in den State rein gepackt, sondern hier einfach ein Attribut: `Timeline timeline` gemacht.